src/main/java/net/teumteum/alert/domain/AlertRepository.java

```diff
@@ -1,7 +1,15 @@
 package net.teumteum.alert.domain;
 
+import java.time.Instant;
+import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface AlertRepository extends JpaRepository<Alert, Long> {
 
+    List<Alert> findAllByUserId(Long userId);
+
+    @Query("select a from alert as a where a.createdAt <= :createdAt")
+    List<Alert> findAll(@Param("createdAt") Instant createdAt);
```

Comment:
```suggestion
    List<Alert> findAllByAfterTime(@Param("createdAt") Instant createdAt);
```
다른 이름으로라도 명시해주면 좋을 것 같슴다

Reply:
[9fdf2cd](https://github.com/depromeet/teum-teum-server/pull/185/commits/9fdf2cdb4005714537efd74b090c44ad6705f934)
반영했습니다